characters/Character.js

```diff
@@ -6,14 +6,23 @@ const Character = mongoose.Schema({
   edited: Date,
   created: Date,
   gender: String,
-  height: String,
+  height: {
+    Number,
+    validate: customHeightStringValidator,
```

Comment:
when using: customHeightStringValidator consider that you have to make that function somewhere inside this file. I do not see it added anywhere. However for height since it is a number. You can just add a min and max property for validation.